examples/franka_blockstacking_example.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import argparse
+
+import numpy as np
+from display_utils import display_rgb
+
+import habitat
+from habitat.core.logging import logger
+
+SENSOR_KEY = "rgb"
+CONFIG_FILE = "configs/tasks/franka_point.yaml"
+BLOCK_LOC_RANGE = 2.0
+BLOCK_LOC_CENTER = [-5.0, 0.1, 8.5]
+
+
+def example(render):
+    # Note: Use with for the example testing, doesn't need to be like this on the README
+
+    with habitat.Env(config=habitat.get_config(CONFIG_FILE)) as env:
+        logger.info("Environment creation successful")
+
+        obj_templates_mgr = env.sim.get_object_template_manager()
+        rigid_obj_mgr = env.sim.get_rigid_object_manager()
+
+        cube_handle = obj_templates_mgr.get_template_handles("cubeSolid")[0]
+        cube_template_cpy = obj_templates_mgr.get_template_by_handle(
+            cube_handle
+        )
+        cube_template_cpy.scale = np.array([0.2, 0.2, 0.2])
```

Comment:
To keep example clean let's use const `BLOCK_SCALE= 0.2`
```suggestion
        cube_template_cpy.scale = np.array([BLOCK_SCALE * 3])
```